root/layout/components/Footer.js

```diff
@@ -9,87 +9,568 @@
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <div className="image me-1">
+                  <img
+                    alt="BookBrainz"
+                    height="24"
+                    src="../../static/images/meb-icons/BookBrainz.svg"
+                    width="24"
+                  />
+                </div>
+                <a
+                  className="fw-bold fs-4"
+                  href="https://bookbrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('BookBrainz')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <div className="image me-1">
+                  <img
+                    alt="AcousticBrainz"
+                    height="24"
+                    src="../../static/images/meb-icons/AcousticBrainz.svg"
+                    width="24"
+                  />
+                </div>
+                <a
+                  className="fw-bold fs-4"
+                  href="https://acousticbrainz.org/"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('AcousticBrainz')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <div className="image me-1">
+                  <img
+                    alt="CoverArtArchive"
+                    height="24"
+                    src="../../static/images/meb-icons/CoverArtArchive.svg"
+                    width="24"
+                  />
+                </div>
+                <a
+                  className="fw-bold fs-4"
+                  href="https://coverartarchive.org"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Cover Art Archive')}
+                </a>
+              </li>
 
-        {nonEmpty(stash.last_replication_date) ? (
-          <>
-            <br />
-            {texp.l('Last replication packet received at {datetime}', {
-              datetime: $c.user
-                ? formatUserDate($c, stash.last_replication_date)
-                : stash.last_replication_date,
-            })}
-          </>
-        ) : null}
-      </p>
-    </div>
+            </ul>
+          </div>
+          <div className="col-sm-12 col-md-2">
+            <h3 className="fs-2 fw-bold color-black">
+              {l('Join Us')}
```

Comment:
I wonder if this shouldn't come before "Fellow Projects," because this is more relevant to MusicBrainz.

Reply:
I have moved the Join Us column before fellow projects and shifted the Useful Links column to the last which solves this problem!